core/src/main/java/smile/vq/SOM.java

```diff
@@ -391,6 +391,19 @@ public SOM(double[][] data, int width, int height) {
     }
 
     /**
+     * Returns the cluster labels for each Neuron.
+     */
+    public int[][] clabels() {
```

Comment:
Can you please rename the function getClusterLabel, which is used by other clustering algorithms?